### ISSUE TITLE
Cannot find OverrideUnityActivity class

### ISSUE THREAD

**Issue by aornano** [Unity-Technologies/uaal-example#25]
Following the instruction , this project crash with this error:
```
/Users/admin/Desktop/uaal-example-master/NativeAndroidApp/app/src/main/java/com/unity/mynativeapp/MainUnityActivity.java:10: error: package com.company.product does not exist
import com.company.product.OverrideUnityActivity;
```

**Comment by carscan** [comment]
please make sure you have copied Android plugin files from UnityProject(/Git/uaal-example/UnityProject/Assets/Plugins/Android) in your own Unity project file. 
Then export your unity project and follow the rest of the process provided in the tutorial.

**Comment by CristianContre** [comment]
> please make sure you have copied Android plugin files from UnityProject(/Git/uaal-example/UnityProject/Assets/Plugins/Android) in your own Unity project file.
> Then export your unity project and follow the rest of the process provided in the tutorial.

Where do I have to copy them?

**Comment by marcspraragen** [comment]
The OverrideUnityActivity can be in just the /assets folder of Android project, right? That's where it would get copied to?

**Comment by johngoren** [comment]
Yeah, so far it seems like the only plugin file I needed for this was a plain old .java superclass file.
